stile/hsc/base_tasks.py

```diff
@@ -329,13 +329,10 @@ def generateColumns(self, dataRef, catalog, mask_tuple, raw_cols, extra_col_dict
 
         # offset for (x,y) if extra_col_dict has a column 'CCD'. Currently getMm() returns values
         # in pixel. When the pipeline is updated, we should update this line as well.
-        if dataRef.dataId.has_key('ccd') and extra_col_dict.has_key(
-            'CCD')and ('x' in raw_cols or 'y' in raw_cols):
-            xy0 =  cameraGeomUtils.findCcd(dataRef.getButler().mapper.camera, cameraGeom.Id(
-                    dataRef.dataId.get('ccd'))
-                                           ).getPositionFromPixel(afwGeom.PointD(0., 0.)).getMm()
-        else:
-            xy0 = None
+        xy0 =  cameraGeomUtils.findCcd(dataRef.getButler().mapper.camera, cameraGeom.Id(
```

Comment:
Is there a reason to make this a ternary operator, out of curiosity?  (Also one of the lines in this expression is longer than our 100 character limit.)